### PR TITLE
feat(shell): support no color mode

### LINKED ISSUE #131: Color escape codes are visible
I'm using the kitty terminal and when I write `MATCH (d:D` it turns to `MATCH ([33md:D`. Is it possible to disable colors in the shell?

### PR DESCRIPTION
**Pull request by naorpeled**
## TODO:
- [x] make sure returned json objects are still formatted, just without color
- [x] add and fix tests (TestSyntaxHighlighterPaint is the only one left to fix)
- [x] cleanup

## Linked issue
closes #131

### CODE REVIEW

File: cmd/cyphernetes/main_test.go [hunk 1]
```diff
@@ -92,7 +92,9 @@ Flags:
 Global Flags:
   -A, --all-namespaces     Query all namespaces
   -l, --loglevel string    The log level to use (debug, info, warn, error, fatal, panic) (default "info")
-  -n, --namespace string   The namespace to query against (default "default")`
+  -n, --namespace string   The namespace to query against (default "default")
+      --no-color           Turn off color formatting
+  `
```
Comment (AvitalTamir) [inline cmd/cyphernetes/main_test.go:97]: the line break here is redundant (output remains the same) and I think inconsistent

**Comment by AvitalTamir** [comment]
Hey I know it's still in draft but I checked it out and looks cool! Works as advertised 😂

**Comment by naorpeled** [comment]
@AvitalTamir still need to fix some tests and add some tests,
will try to get to it later today, worst case will do it tomorrow evening 🙏

File: cmd/cyphernetes/query.go [hunk 2]
```diff
@@ -69,5 +69,5 @@ func runQuery(args []string, w io.Writer) {
 
 func init() {
 	rootCmd.AddCommand(queryCmd)
-	queryCmd.PersistentFlags().BoolVarP(&disableColorJsonOutput, "raw-output", "r", false, "Disable colorized JSON output")
+	queryCmd.PersistentFlags().BoolVarP(&returnUnformattedJsonOutput, "raw-output", "r", false, "Disable colorized JSON output")
```
Comment (naorpeled) [inline cmd/cyphernetes/query.go:72]: @AvitalTamir should I also update the description here?

**Comment by naorpeled** [comment]
@AvitalTamir still got some issues to resolve, hopefully will be able to boost this significantly tomorrow 🙏

**Comment by AvitalTamir** [comment]
All good, let me know if you need any help bringing this to the finish line

**Comment by naorpeled** [comment]
> All good, let me know if you need any help bringing this to the finish line

Actually if you'll have a bit of time to sync over Zoom tomorrow that'd be awesome

**Comment by AvitalTamir** [comment]
Let's do it

**Comment by naorpeled** [comment]
> Let's do it

Would 18:30 work for you?

**Comment by AvitalTamir** [comment]
Yea, DMing you

**Comment by naorpeled** [comment]
TODO (in a separate PR):
- [ ] don't format `AS` as uppercase if part of a json path (e.g. bla.as becomes bla.AS) - this is the current behavior before my change.
- [ ] support uppercasing and coloring the `return` keyword even if there's nothing after it (the other keywords work like that so I guess it'd be nice to have the same behavior there as well)

File: cmd/cyphernetes/shell.go [hunk 3]
```diff
@@ -626,6 +639,10 @@ func init() {
 	if namespace != "" && namespace != "default" {
 		parser.Namespace = namespace
 	}
+
+	if _, exists := os.LookupEnv("NO_COLOR"); exists {
```
Comment (AvitalTamir) [inline cmd/cyphernetes/shell.go:643]: Yes! 🙌

**Review by AvitalTamir — APPROVED** [review]
@naorpeled thank you so much for putting in the effort. Great work.
Are we ok now with the tests in `main_test.go` or you still experiencing failures locally?
Locally they pass for me, and I see CI is green as well.
Let me know, if we're good I'll merge :)

**Comment by naorpeled** [comment]
> @naorpeled thank you so much for putting in the effort. Great work. Are we ok now with the tests in `main_test.go` or you still experiencing failures locally? Locally they pass for me, and I see CI is green as well. Let me know, if we're good I'll merge :)

They seem to be passing now, I think it was an issue with my local setup, did some cleanup and now it seems to be passing consistently.

I believe we can merge this now :)

**Review by AvitalTamir — APPROVED** [review]

**Pull request merged**